apps/journeys-admin/src/components/Team/UserTeamInviteForm/UserTeamInviteForm.tsx

```diff
@@ -104,13 +106,18 @@ export function UserTeamInviteForm({
               label={t('Email')}
               name="email"
               fullWidth
+              disabled={role !== 'manager'}
               variant="filled"
               value={values.email}
               onChange={handleChange}
               onBlur={handleBlur}
               error={errors.email != null && touched.email != null}
               helperText={
-                touched?.email != null && errors.email != null && errors.email
+                role !== 'manager'
+                  ? t('Only a Manager can invite new members to the team')
```

Comment:
```suggestion
                  ? t('Only a manager can invite new members to the team')
```

be sure to run extract-translations